Service/Request/Visitor/EndpointConfiguration.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace Auto1\ServiceAPIClientBundle\Service\Request\Visitor;
+
+/**
+ * Class EndpointConfiguration.
+ */
+class EndpointConfiguration
```

Comment:
Should it be `EndpointConfiguration`, since we have already the same class in this package.
It could be quite confusing.